### PR TITLE
Cookies

### PR DESCRIPTION
**Pull request by AlirieGray**
Added react-cookie npm module,
when the user logs in successfully, the token is stored as a cookie in the browser. Then, in the home component, we check for the cookie. If the cookie exists, we show the dashboard component. Otherwise, redirect to login

### CODE REVIEW

File: src/components/Home.js [hunk 1]
```diff
@@ -24,11 +25,12 @@ class Home extends Component {
 
   componentWillMount() {
     const { cookies } = this.props;
+
     if (!this.state.authChecked) {
       //this.checkAuth()
       // check if the cookie is stored
       this.setState({
-        isLoggedIn: cookies.get("RubricsApp"),
+        isLoggedIn: cookies.get("RubricsApp") !== "",
```
Comment (Jeffchiucp) [inline src/components/Home.js]: I think we also need to also if it is equal to "undefined".
It's good that we also check it's empty string right now

Comment (Jeffchiucp) [reply src/components/Home.js]: We can make the changes in the next code changes

File: src/components/Home.js [hunk 2]
```diff
@@ -59,8 +61,8 @@ class Home extends Component {
     // TODO:add activeClassName for when the link is selected
     return (
       <Route exact path="/" render={() => (
-          this.isLoggedIn ? (
-            <Sidebar/>
+          this.state.isLoggedIn ? (
+            <Dashboard />
           ) : (
             <Redirect to="/login"/>
```
Comment (Jeffchiucp) [inline src/components/Home.js]: I don't think redirect works at home component.
Please make those change `            this.props.history.push('/login')`

Comment (Jeffchiucp) [reply src/components/Home.js]: replace `<Redirect to="/login"/>` with  `           this.props.history.push('/login')`

Comment (Jeffchiucp) [reply src/components/Home.js]: This is a bug. And we identified bugs on the login url

Comment (Jeffchiucp) [reply src/components/Home.js]: We will need to fix this before our next demo... creating a new branch `Fix-Login`

Comment (tlambrou) [reply src/components/Home.js]: This is not how the structure of components work.  Please see the Frontend Component Architecture document.

File: src/components/Home.js (hunk 1 above)
Comment (tlambrou) [reply src/components/Home.js]: We still need to check for undefined

**Pull request merged**